etl/steps/data/garden/health/2022-12-28/karlinsky.meta.yml

```diff
@@ -0,0 +1,50 @@
+dataset:
+  sources:
+  #   -
+  #     name: Maddison Project Database 2020 (Bolt and van Zanden, 2020)
+  #     published_by: Bolt, Jutta and Jan Luiten van Zanden (2020), “Maddison style estimates of the evolution of the world economy. A new 2020 update“.
+  #     publisher_source: |
+  #       The Maddison Project Database is based on the work of many researchers that have produced estimates of...
+
+tables:
+  deaths_karlinsky:
+    title: International Completeness of Death Registration 2015-2019 – Karlinsky (2021)
+    description: >
+      Contains all metrics used in the International Completeness of Death Registration 2015-2019 paper.
+    variables:
+      # Life expectancy
+      reg_deaths:
+        title: Registered deaths
+        unit: "deaths"
+        description: >
+          The number of deaths registered in the vital registration system for the country-year. NA denotes no registered deaths data was obtained.
```

Comment:
Do we actually have verbose "NA" in the data? I guess we shouldn't. And if we don't, then this should be removed from th description.

Reply:
We have NaNs, as these are present ins some of the variables. I think this is fine and also expected, as some country-years are missing data.

Reply:
Sure, I'm just wondering if they will be seen as "NA", as the description says. I guess they will not, they will simply be missing. If so, then it makes more sense to say that some country-years may not have data because there were no registered deaths.

Reply:
I see! That makes sense. To some extent this is the metadata for Garden, so I think talking about `NaN`s is fine. However, this is also what we currently show in Grapher, where `NaN`s are not visible. I think your description is compatible with both channels, thanks!